src/api/services/notifications/helpers.py

```diff
@@ -29,6 +29,13 @@ def create_celery_flask(environment=os.getenv('APP_SETTINGS', 'Production')):
     """
     app = Flask(__name__)
```

Comment:
We could make this its own API, with a post URL to decouple it completely from the main app. `<not critical>`

Reply:
Good point, but I'd say the notifications work independently of the main app as it is. The celery tasks decorator is an API on its own so creating an emails `POST` endpoint would be akin to adding another API layer on top of Celery's.